test/FsAutoComplete.Tests.Lsp/Utils/CursorbasedTests.fs

```diff
@@ -114,7 +114,7 @@ module CodeFix =
     (beforeWithCursor: string)
     (validateDiagnostics: Diagnostic[] -> unit)
     (chooseFix: ChooseFix)
-    (expected: ExpectedResult)
+    (expected: unit -> ExpectedResult)
```

Comment:
this was to delay work done at test-app-startup time - I wanted to debug the comparison and _every_ test was calling this.